.readthedocs.yml

```diff
@@ -4,7 +4,7 @@ build:
   image: latest
 
 python:
-  version: 3.7
+  version: 3.9
```

Comment:
Last time I tried, RTD won't build in Python 3.9 and I had to fall back to 3.8. 🤷

Reply:
Ugh, I just now saw this.  Let's see.  🤞

Reply:
Indeed, 3.9 is not allowed:  "Invalid "python.version": expected one of (2, 2.7, 3, 3.5, 3.6, 3.7, 3.8, pypy3.5), got 3.9"

Reply:
Maybe @astrojuanlu can tell us when RTD supports Python 3.9. 😸

Reply:
It supports 3.9 and 3.10 since October! :) https://blog.readthedocs.com/new-build-specification/